app/lib/contacts/distributed_lock.rb

```diff
@@ -1,3 +1,4 @@
+# :nocov:
```

Comment:
I'm a bit confused by this commit, where is this logic tested?

Reply:
This code is used only by the [import organisations rake task](https://github.com/alphagov/contacts-admin/blob/main/lib/tasks/organisations.rake), which creates this distributed lock and calls [this method](https://github.com/alphagov/contacts-admin/blob/main/app/tasks/import_organisations.rb). That method  is [tested](https://github.com/alphagov/contacts-admin/blob/main/spec/tasks/import_organisations_spec.rb) separately, so it's only the lock itself that's not; but testing the lock feels like it would mean stubbing/mocking Redis, which leaves little of value left to test.

Reply:
Could probably easily adjust the above into a test on DistributedLock class itself too